docs/fundamentals/integrations-overview.md

```diff
@@ -14,19 +14,119 @@ ms.topic: conceptual
 
 ## Integration responsibilities
 
-There are two sides to integrations in .NET Aspire, each with a different responsibility:
```

Comment:
Can we also tweak the sample at the bottom of this document to show both the hosting integration and the client integration?

Reply:
I was thinking about removing that sample in this doc, doesn't really feel like the right place to have a sample. Thoughts?

Reply:
Agreed.